`make verify` is failing due to golint failure

```
➜ make verify
./scripts/check_gofmt.sh
./scripts/check_golint.sh
/Users/mho/go/src/github.com/gomods/athens/pkg/eventlog/fs/pointer_registry.go:77:2: redundant if ...; err != nil check, just return error instead.
Found 1 lint suggestions; failing.
make: *** [verify] Error 1
```

--- a/pkg/eventlog/fs/pointer_registry.go
+++ b/pkg/eventlog/fs/pointer_registry.go
@@ -74,9 +74,5 @@ func (r *Registry) SetPointer(deploymentID, pointer string) error {
 	}
 
 	enc := json.NewEncoder(f)
-	if err := enc.Encode(&data); err != nil {
-		return err
-	}
-
-	return nil
+	return enc.Encode(&data)
 }

